### PR TITLE
Product details

### PR DESCRIPTION
**Pull request by faridatarek**

### CODE REVIEW

File: lib/core/widgets/custom_appbar.dart
Comment (OmarAbdelgilil) [inline lib/core/widgets/custom_appbar.dart:1]: why still using this appbar ?

File: lib/core/widgets/custom_button.dart
Comment (Abdalla-Eldaly) [inline lib/core/widgets/custom_button.dart:1]: use AppTextStyles

File: lib/features/home/presentation/product_details_screen/product_details_view/product_details_screen.dart
Comment (OmarAbdelgilil) [inline lib/features/home/presentation/product_details_screen/product_details_view/product_details_screen.dart:1]: still using builder ?

**Review by Abdalla-Eldaly — APPROVED** [review]
great job

File: lib/features/home/presentation/product_widget/product_view_model/product_view_model.dart
Comment (OmarAbdelgilil) [inline lib/features/home/presentation/product_widget/product_view_model/product_view_model.dart:1]: this wasn't your task I think

**Review by OmarAbdelgilil — CHANGES_REQUESTED** [review]
take a look at the comments - too much conflicts are u sure u pulled dev branch

**Review by Menna1311 — APPROVED** [review]
great job

**Review by OmarAbdelgilil — APPROVED** [review]
Well done ! 👏

**Pull request merged**